.pre-commit-hooks.yaml

```diff
@@ -6,7 +6,7 @@
   args: [-w, -s]
   types: [shell]
   exclude_types: [csh, tcsh, zsh]
-  stages: [commit, merge-commit, push, manual]
+  stages: [pre-commit, pre-merge-commit, pre-push, manual]
```

Comment:
To err on the side of caution, let's add a minimum pre-commit version to all these changes, like in https://github.com/scop/pre-commit-perlcritic/commit/d2b81b4977a45d6cbc651bdfb5a713832a1d98f2